src/Block/Service/AbstractCategoriesBlockService.php

```diff
@@ -31,6 +31,8 @@
 use Symfony\Component\OptionsResolver\OptionsResolver;
 use Twig\Environment;
 
+use function is_string;
```

Comment:
we normally not use this on Sonata packages

Reply:
solved